src/metabase/plugins.clj

```diff
@@ -112,12 +114,16 @@
   same plugins from the uberjar. This is needed because some plugin manifests define driver methods and the like that
   aren't defined elsewhere."
     []
-    ;; TODO - this should probably do an actual search in case we ever add any additional directories
-    (doseq [path  (files/files-seq (files/get-path "modules/drivers/"))
-            :let  [manifest-path (files/get-path (str path) "/resources/metabase-plugin.yaml")]
-            :when (files/exists? manifest-path)]
-      (log/info (trs "Loading local plugin manifest at {0}" (str manifest-path)))
-      (load-local-plugin-manifest! manifest-path))))
+    (doseq [^File file (classpath/system-classpath)
+            :when      (and (.isDirectory file)
+                            (not (.isHidden file))
+                            (str/includes? (str file) "modules/drivers")
+                            (str/ends-with? (str file) "resources"))
+            :let       [manifest-file (io/file file "metabase-plugin.yaml")]
+            :when      (.exists manifest-file)
+            :let       [info (yaml/parse-string (slurp manifest-file))]]
+      (log/info (trs "Loading local plugin manifest at {0}" (str manifest-file)))
+      (load-local-plugin-manifest! manifest-file))))
```

Comment:
This is the tweak to make sure we only initialize local drivers from `modules/drivers` if they're actually on the classpath